### PR TITLE
Added new chart on the metrics page, new component built using d3 

### PR DESCRIPTION
**Pull request by Ab-hay**
*Issue #, if available:*

*Description of changes:*
Metrics page no longer has <li> text fields, replaced them with a beautiful donut chart.

### CODE REVIEW

File: browser/components/MetricsLists.tsx [hunk 1]
```diff
@@ -14,6 +14,8 @@
 import * as React from 'react';
 import SimpleLineChart from '../components/SimpleLineChart';
 
+import RadzChart from '../components/RadzChart';
```
Comment (jamesiri) [inline browser/components/MetricsLists.tsx]: Lets use a more descriptive name for this component.

File: browser/components/RadzChart.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      .attr('transform', function(d) {
+        const pos = outerArc.centroid(d);
+        pos[0] = radius * 0.95 * (midAngle(d) < Math.PI ? 1 : -1);
+        return 'translate(' + pos + ')';
+      })
+      .style('text-anchor', function(d) {
+        return midAngle(d) < Math.PI ? 'start' : 'end';
+      });
+
+    const g = svg
```
Comment (jamesiri) [inline browser/components/RadzChart.tsx]: This is not descriptive of what the parameter contains. Please fix.

Comment (Ab-hay) [reply browser/components/RadzChart.tsx]: g is a sub-element of Svg. its pretty cool, you can apply same transformation all the elements inside g tag. here's more: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g

File: browser/components/MetricsLists.tsx (hunk 1 above)
Comment (Ab-hay) [reply browser/components/MetricsLists.tsx]: what do you think about DonutChart.tsx ?

File: browser/components/RadzChart.tsx (hunk 2 above)
Comment (jpeddicord) [reply browser/components/RadzChart.tsx]: A comment here might be a good compromise, since `g` is the actual SVG tag name for a group.

**Review by jpeddicord — APPROVED** [review]
Nice change! I'm excited to see more pretty charts on the front page. :)

**Pull request merged**